extensions/blocks/podcast-player/attributes.js

```diff
@@ -20,4 +24,22 @@ export default {
 		type: 'boolean',
 		default: true,
 	},
+	primaryColor: {
+		type: 'string',
```

Comment:
Do you think we could add our `colorValidator` utility as a `validator` for those colors, if we always expect hex values?

Reply:
`primaryColor` is a string. We could add hex values for the custom ones, though. Let me take a look at how to do it. Good advice.